### DISCUSSION TITLE
Unsubscribe please Help

### DISCUSSION THREAD

**Discussion by Jeramy-Babb** [iptv-org/community#1027] [Q&A]
Hi everybody it's me.  I like this group and I am using but I made a little mistake of subscribing and now I have to say I am getting more mail than ever before since I was in the chat.  Just 1 after another.  I unsubscribe and the mail keeps coming.  Please help take me of the list.  Thank you.

**Comment by freearhey** [comment]
https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications
